tests/unit_tests/test_variance.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def variance_process_code():
+    return load_process_code("variance")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({"data": [-1, 1, 3, None]}, 4),
```

Comment:
Is this correct? I would expect it to be 2.6666666666666665
average is 1, number of elements is 3, sum of squares is 8, result should be 8/3